README.md

```diff
@@ -124,6 +124,14 @@ Enable streaming responses from the Vectara API. Defaults to true.
 The language the response should be in. Defaults to "eng" for English.
 See our [types](src/types.ts) for more information on supported language values.
 
+##### `factualConsistencyScore` (optional)
```

Comment:
Hmmm now that I think about it, what do you think about naming this `calculateFactualConsistencyScore` for clarity? As it is, the name gives the idea that this prop accepts an actual FCS instead of a directive to calculate it.

Reply:
Good catch! I actually got this wrong -- it should be `enableFactualConsistencyScore`.